sonar-batch/src/main/java/org/sonar/batch/phases/DecoratorsExecutor.java

```diff
@@ -51,15 +50,12 @@
   private final CoverageExclusions coverageFilter;
   private final MeasureCache measureCache;
   private final MetricFinder metricFinder;
-  private final DuplicationCache duplicationCache;
   private final AnalysisMode analysisMode;
 
-  public DecoratorsExecutor(BatchExtensionDictionnary batchExtDictionnary,
-    Project project, SonarIndex index, EventBus eventBus, CoverageExclusions coverageFilter, MeasureCache measureCache, MetricFinder metricFinder,
-    DuplicationCache duplicationCache, AnalysisMode analysisMode) {
+  public DecoratorsExecutor(BatchExtensionDictionnary batchExtDictionnary, Project project, SonarIndex index, EventBus eventBus, CoverageExclusions coverageFilter,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/henryju/image-hosting/master/major.png)Method has 8 parameters, which is greater than 7 authorized.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00107)